src/interactions/panZoomInteraction.ts

```diff
@@ -248,6 +366,9 @@ export module Interactions {
      * @returns {Interactions.PanZoom} The calling PanZoom Interaction.
      */
     public addXScale(xScale: QuantitativeScale<any>) {
+      if (this._nonLinearScaleWithExtents(xScale)) {
```

Comment:
This isn't going to trigger properly. New `Scale`s won't have any extent limits attached, so there won't be any warnings. At the same time, attempting to put an extent limit on the `Scale` won't produce warnings.